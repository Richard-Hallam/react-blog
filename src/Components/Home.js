
function Home(){
    return(
        <p class="text-9xl font-extrabold">HOMEPAGE</p>
    )
}

export default Home;