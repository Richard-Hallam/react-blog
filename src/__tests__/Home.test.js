import React from "react";
import { getByText, render, screen } from '@testing-library/react';
import Home from "../Components/Home";

describe("Projects", () => {
    const { asFragment } = render(<Home />);

    it("renders correctly", ()=>{
        expect(asFragment()).toMatchSnapshot();
    });


});