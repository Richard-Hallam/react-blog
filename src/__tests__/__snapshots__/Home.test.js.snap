// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders correctly 1`] = `
<DocumentFragment>
  <p
    class="text-9xl font-extrabold"
  >
    HOMEPAGE
  </p>
</DocumentFragment>
`;
